src/components/QuickActions.vue

```diff
@@ -26,7 +74,7 @@ const logout = (quickAction: any) => {
         class="d-flex flex-column align-center"
       >
         <BaseButton
-          @click="logout(quickAction)"
+          @click="quickAction.action ? quickAction.action() : null"
```

Comment:
Why there a need for ternary operator?  
Ideally each action button should have an action associated to it. It can be null or empty string for now.